terraform/services/quicksight/main.tf

```diff
@@ -45,4 +48,10 @@ module "dpc_insights_data" {
 module "dpc_insights_athena" {
   source = "../../modules/bucket"
   name   = local.dpc_athena_s3_name
+}
+
+resource "aws_s3_object" "folder" {
+  bucket       = module.dpc_insights_athena.id
+  content_type = "application/x-directory"
+  key          = local.dpc_athena_results_folder_key
```

Comment:
I'm a little unclear as to why this resource is needed. Is there an error if the output_location is supplied without this?

Reply:
TF definitely argued when I didn't have a specific Result_Configuration folder of this object type for the Athena Workgroup resource; I pulled over the comparable config from BFD-Insights

Reply:
Still seems odd to me. I don't see any docs about the need to define an x-directory object in S3 for Athena. But it's fine to leave it in for now.